proto/process/agent.proto

```diff
@@ -97,6 +97,9 @@ message CollectorConnections {
 	repeated string domains = 30;
 
 	repeated Route routes = 31;
+
+	map<string, int32> resolvedRoutes = 34;
```

Comment:
Is the key here IP? Why not embed this directly into each Connection?

Reply:
Actually this map looks wrong; it is key'ing off of destination ip, when it should be key'ing off of (destination, subnet-id). Embedding this info in the connection would then make more sense. Let me change this.